addon/helpers/compact.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+const {
+  Helper,
+  get,
+  set,
+  observer,
+  isPresent,
+  isArray,
+  A: emberArray,
+  computed: { filter }
+} = Ember;
+
+export default Helper.extend({
+  compute([array]) {
+    if (!isArray(array)) {
+      return emberArray([array]);
+    }
+
+    set(this, 'array', array);
+
+    return get(this, 'content');
+  },
+
+  array: null,
```

Comment:
Please move this to the L15, we prefer to group props first, then computed/observers